source/_docs/backend/database.markdown

```diff
@@ -3,9 +3,9 @@ title: "Database"
 description: "Details about the database used by Home Assistant."
 ---
 
-Database is used in by Home Assistant as history and tracker only, to store the events and its parameters. The default database used by Home Assistant is [SQLite](https://www.sqlite.org/), and the database file is stored in your [configuration directory](/getting-started/configuration/) (e.g., `<path to config dir>/home-assistant_v2.db`). If you prefer to run a database server (e.g.,  PostgreSQL), use the [`recorder` component](/integrations/recorder/).
+Home Assistant uses databases to store events and parametersis for history and tracking. The default database used is [SQLite](https://www.sqlite.org/) and the database file is stored in your [configuration directory](/getting-started/configuration/) (e.g., `<path to config dir>/home-assistant_v2.db`); however, other databases can be used. If you prefer to run a database server (e.g.,  PostgreSQL), use the [`recorder` component](/integrations/recorder/).
```

Comment:
Home Assistant uses 1 database for this purpose, so using plural seems wrong in this case.

Reply:
Thanks for the clarification. I'll be happy yo change "uses databases" to "uses a database", but I'm not sure how to do that yet (i'm looking at Hacktoberfest's beginners resources to find out)

Reply:
![image](https://user-images.githubusercontent.com/195327/95052751-c3fcb100-06ef-11eb-8a23-1b737be422e5.png)

Go to the "Files changed" tab -> hit the `...` menu on the file you want to edit -> Edit file